examples/computer_vision/vision_transformer/det_callback.py

```diff
@@ -0,0 +1,187 @@
+import typing
+
+from transformers import (
+    TrainerCallback,
+    TrainerState,
+    TrainerControl,
+    TrainingArguments,
+)
+
+import os
+import determined as det
+import logging
+
+import weakref
+import importlib.util
+
+
+class DetCallback(TrainerCallback):
```

Comment:
instead of having this as an example, i wonder if we should have module in harness with integrations via callbacks.